src/data/morefaq.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+      "question": "What is GlobeHoppin?",
+      "answer": "GlobeHoppin is a cutting-edge platform designed for travel enthusiasts. It offers a comprehensive suite of tools for discovering unique travel resources, connecting with fellow adventurers, and planning unforgettable journeys across the globe."
+    },
+    {
+      "question": "How can I contact support?",
+      "answer": "Our dedicated support team is always ready to assist you. You can reach us through our Contact page, where you'll find options for email support, live chat, and phone assistance. We aim to respond to all inquiries within 24 hours."
+    },
+    {
```

Comment:
You can merge both faqData and moreFaq into into single fine. On home just display top 5.